assets/admin/editor-wizard/helpers.js

```diff
@@ -188,3 +189,30 @@ export const useHideEditorWizardUpsell = () => {
 		! senseiProExtension || senseiProExtension.is_activated === true
 	);
 };
+
+/**
+ * Hook to get the site slug.
+ *
+ * @return {string} Site slug.
+ */
+export const useSiteSlug = () => {
```

Comment:
I don't think the `editor-wizard` would be the best place to add this since we should use it in some different parts.

Maybe it was a confusion because of the screenshot I added to the issue. That was supposed to be only an example, and we have other upsells in the plugin that I didn't map as part of the issue. Sorry! 😜

So maybe we could add these functions in some place like `assets/admin`?

Reply:
I think the best way to find all the places with upsells is through a search in all the codebase for `https://senseilms.com/sensei-pro`.

Reply:
> Maybe it was a confusion because of the screenshot I added to the issue. That was supposed to be only an example, and we have other upsells in the plugin that I didn't map as part of the issue. Sorry! 😜

No problem. I'm going to update the logic and cover all upsell cases. 👌